### ISSUE TITLE
feat: add grouped select support when binding observables

### ISSUE THREAD

**Issue by juristr** [formly-js/ngx-formly#775]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request
```
See #772.

**Comment by juristr** [comment]
@aitboudad you can assign me to this issue

**Comment by chaosLegacy** [comment]
Hi,
I'l having a weird issue with select group that close automatically without doing anything, what my code do is generating fields dynamically according to some parameter i'm passing for example here i'm generating a grouped select:
`if (attribute.attributeType === 'FUNCTION') {

          _.extend(genericField,
            {
              type: 'select',
              key: 'champs#' + attribute.id,
              defaultValue: defaultValue || '',
              templateOptions: {
                label: attribute.name,
                options: this.userListFunctions,
                valueProp: 'code',
                labelProp: 'title',
                groupProp: 'category',
                required: true,
              }
            });
        }`
the input is displaying fine and it filled with the values i wanted everything is okay but as soon as i click on the select to choose a value it close automatically what i notice is when i remove the `groupProp` the select is working fine i also added a group property inside my list `userListFunctions` but it keep closing, PS: a keyboard event like space stop the select from closing now i'm trying to simulate the keyboard space when i click on the select pretty much stupid i know but i couldn't find any hack to make it work properly
many thanks to all of you guys great library. 

<img width="796" alt="screen shot 2018-04-26 at 10 45 37 am" src="https://user-images.githubusercontent.com/11313734/39298955-7e357672-493f-11e8-9b4a-2861031280d7.png">
<img width="797" alt="screen shot 2018-04-26 at 10 45 48 am" src="https://user-images.githubusercontent.com/11313734/39298956-7e50bc52-493f-11e8-8935-85d7f5a0bb5b.png">
<img width="798" alt="screen shot 2018-04-26 at 10 45 58 am" src="https://user-images.githubusercontent.com/11313734/39298957-7e6e5af0-493f-11e8-9a22-6664307cf712.png">
<img width="815" alt="screen shot 2018-04-26 at 10 46 27 am" src="https://user-images.githubusercontent.com/11313734/39298960-7e8bbffa-493f-11e8-9563-047dcff29595.png">

**Comment by juristr** [comment]
Would it be possible to have an isolated StackBlitz reproduction of this behaviour? That'd be a lot of help, so I can quickly give it a look when I have a couple of minutes of time 👍

**Comment by aitboudad** [comment]
solved by #956 while trying to fix #953
